controllers/news.controllers.js

```diff
@@ -0,0 +1,20 @@
+const{fetchAllTopics,fetchAllEndpoints} =require("../models/news.models")
+
+exports.getTopics=(req,res,next)=>{
+    fetchAllTopics().then((topics)=>{
+        return res.status(200).send({topics});
+    })
+    .catch((err)=>{
+        next(err);
+    })
+}
+
+exports.getEndpoints=(req,res,next)=>{
+    fetchAllEndpoints().then((endpoints)=>{
+        console.log(endpoints,'contr');
```

Comment:
Don't forget to remove console logs once you've finished developing on an endpoint 🧹